# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/public/experimental/TypedData.h
# Commit: df0c1ee4d0b0
# Full Hash: df0c1ee4d0b0f11ea3149daf4b2a7d2813de6fea
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-09-09 15:22:41
# Regressor Bug: 1720422
# File Overlap Count: 2
# Description:
#   Bug 1720422 - Fill in the rest of the ArrayBuffer+ArrayBufferView hierarchy, and base more ArrayBufferViewObject public APIs off of new API r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119857
# ==============================================================================

diff -r e0d16e395a5f -r df0c1ee4d0b0 js/public/experimental/TypedData.h
--- a/js/public/experimental/TypedData.h	Wed Sep 08 23:46:58 2021 +0000
+++ b/js/public/experimental/TypedData.h	Wed Sep 08 23:46:58 2021 +0000
@@ -387,6 +387,7 @@
     }
     return ArrayBuffer(nullptr);
   }
+  static ArrayBuffer unwrap(JSObject* maybeWrapped);
 
   static ArrayBuffer create(JSContext* cx, size_t nbytes);
 